### PR TITLE
Bad import in test_torchinductor and skip torchvision related UT

### PR DESCRIPTION
**Pull request by pragupta**
Fixes https://ontrack-internal.amd.com/browse/SWDEV-444963

### CODE REVIEW

File: test/inductor/test_torchinductor.py [hunk 1]
```diff
@@ -72,6 +72,7 @@
     parametrize,
     skipIfRocm,
     skipIfRocmArch,
+    subtest,
```
Comment (pruthvistony) [inline test/inductor/test_torchinductor.py:75]: Why this is required?

File: test/inductor/test_torchinductor.py [hunk 2]
```diff
@@ -5614,6 +5615,7 @@ def fn(in_ptr0, in_ptr1, in_ptr2):
             ),
         )
 
+    @skipIfRocm
```
Comment (pruthvistony) [inline test/inductor/test_torchinductor.py:5618]: Is there a problem with torchvision compatibility version installed?

File: test/inductor/test_torchinductor.py (hunk 1 above)
Comment (pragupta) [reply test/inductor/test_torchinductor.py:75]: This import was already there, it was accidentally removed by this commit:

https://github.com/ROCm/pytorch/commit/0b08ab6eeffd5d1ad34bf74affc810ae23542992#diff-809c39aeafb3acc92289f42a63e670a8719d4ce5627d5f88820142d80edf8d2aL74

File: test/inductor/test_torchinductor.py (hunk 2 above)
Comment (pragupta) [reply test/inductor/test_torchinductor.py:5618]: It seems that when this test is running, it tries to establish a console. That's why, I skipped it.

**Review by pruthvistony — APPROVED** [review]
Ok to skip this test for now.
Since this is an upstream problem, should we raise to upstream and work on the fix?

**Comment by pragupta** [comment]
Opened an issue in my backlog to track torchvision UT https://github.com/ROCm/frameworks-internal/issues/7549

**Pull request merged**